### PR TITLE
corrected the minor mistakes

### PR DESCRIPTION
**Pull request by nitishk72**
#251 Readme file was having some minor mistakes in English. I had updated them if you find it correct then do merge it. If there is any issue with this then let me know. 

I'll happy to contribute to this great plugin

### CODE REVIEW

**Comment by HofmannZ** [comment]
Thanks for the PR! We're currently working on the new version of the repo on the beta branch, where we split the client into two packages. Each package has its own `README.md` alongside the `README.md` in the root of the project.

Could you try to merge this PR against the `beta` branch?

**Comment by nitishk72** [comment]
Beta branch is not having README instruction

**Comment by micimize** [comment]
@nitishk72 made some minor corrections to your minor corrections - `beta`'s `graphql_flutter` readme is at[`/packages/graphql_flutter/README.md`](https://github.com/zino-app/graphql-flutter/blob/beta/packages/graphql_flutter/README.md), but it's going to be a bit tedious to rebase for this. I'd just manually edit in browser and re-PR

File: README.md [hunk 1]
```diff
@@ -259,7 +259,7 @@ ValueNotifier<GraphQLClient> client = ValueNotifier(
 );
 ```
 
-`dataIdFromObject` is required and has no defaults. Our implementation is similar to apollo's, requiring a function to return a universally unique string or `null`. The predefined `typenameDataIdFromObject` we provide is similar to apollo's default:
+`dataIdFromObject` is required and has no defaults. Our implementation is similar to Apollo's, requiring a function to return a universally unique string or `null`. The predefined `typenameDataIdFromObject` we provide is similar to apollo's default:
```
Comment (ammaratef45) [inline README.md]: @nitishk72 `is similar to apollo's default` Here make it Apollo too.

**Comment by micimize** [comment]
whaaaat the heck. I force pushed master instead of my rebase and it auto closed the PR because there are no changes, so now I can't fix the issue because github revoked my write access

**Comment by micimize** [comment]
Well, that's a moronic anti-feature. I went ahead and just pushed the commit (f0da96bfce27941141130eeef4ba1406ccd9713e). Sorry about that @nitishk72

**Pull request closed without merging**